components/ui/AppCard.vue

```diff
@@ -21,16 +21,18 @@
           />
         </div>
       </header>
-      <div class="app-card__description">
-        <slot />
+      <div>
+        <div class="app-card__description">
+          <slot />
+        </div>
+        <AppCta
+          v-if="to"
+          class="app-card__cta"
+          v-bind="ctaLink"
+          is-wider
+          kind="ghost"
+        />
```

Comment:
Although this is already explained in the commit's description, here is a visual preview of how the card elements would align without this:

![image](https://user-images.githubusercontent.com/22047320/119493138-2cbc9a80-bd60-11eb-9d3a-07cfdfb7d0ef.png)

And this is how the elements get properly aligned after this change:

![image](https://user-images.githubusercontent.com/22047320/119493214-43fb8800-bd60-11eb-9cac-bc2499bfe875.png)